### PR TITLE
convert js hook to ts

### PR DESCRIPTION
**Pull request by nikasepiskveradze**
I added and converted those hooks, useToggle hook missing Github gist and Codesandbox links so will you do it or should i add those links?

### CODE REVIEW

**Comment by gragland** [comment]
Great! I'll test this out soon and I'll add in the gist and codesandbox links.

**Comment by nikasepiskveradze** [comment]
When you finish testing useToggle, please also confirm all those hooks which i have already converted to TS

File: src/pages/useDebounce.md [hunk 1]
```diff
@@ -105,3 +106,103 @@ function useDebounce(value, delay) {
   return debouncedValue;
 }
 ```
+
+```typescript jsx
```
Comment (gragland) [inline src/pages/useDebounce.md]: This should just be `typescript` not `typescript jsx`

**Comment by gragland** [comment]
@nikasepiskveradze I'm actually totally new to TS so my ability to catch mistakes is limited. I'm happy to take your word for it if you think it's good to go though. Maybe just do a quick double-check and then let me know if it looks good to you?

**Comment by nikasepiskveradze** [comment]
> @nikasepiskveradze I'm actually totally new to TS so my ability to catch mistakes is limited. I'm happy to take your word for it if you think it's good to go though. Maybe just do a quick double-check and then let me know if it looks good to you?

@gragland  Thanks for trusting. I double checked and changed minor things, added also Github gist and codesandbox for useToggle hook and i think everything is ready to go you can merge it.

File: src/pages/useDebounce.md (hunk 1 above)
Comment (nikasepiskveradze) [reply src/pages/useDebounce.md]: Fixed

**Pull request merged**